docs/index.rst

```diff
@@ -85,6 +85,24 @@ your ~/.bash_profile::
 
 Note that a functional NetCDF install is required.
 
+Windows specific installation instructions
+------------------------------------------
+
+To enable Parcels to run on a Windows machine, some extra steps have to be taken to install the necessary prerequisite files. Windows must have the following software before it is able to run Parcels:
+* A working distribution of gcc must be installed. With working is meant that gcc an be accessed from the commandline. We suggest Windows users to install the `MinGW Package <www.mingw.org>`_. Do note that to access the gcc compiler which MinGW provides, the user must most likely add the directory path to the 'MinGW\bin' folder where gcc resides.
```

Comment:
I would simply remove the sentence `With working is meant that gcc an be accessed from the commandline.` 

And can we provide a link to a site that gives instructions on how to set the PATH?